matrix-sdk/src/main/java/org/matrix/androidsdk/data/Room.java

```diff
@@ -2125,62 +2125,7 @@ public void sendEvent(final Event event, final ApiCallback<Void> callback) {
 
             // Encrypt the content before sending
             mDataHandler.getCrypto()
-                    .encryptEventContent(contentAsJsonObject, event.getType(), this, new ApiCallback<MXEncryptEventContentResult>() {
```

Comment:
Taking the callback out of this argument list is a good move.  I can't think of any value in having a new ApiCallback in that argument list be defined within it.  That being said, it would be nice to see `getCallback()` itself either get cut down further or moved off into a new file with `getLocalCB()` from [b2d51e90321c2d4f06a2f152727f14f5d42561a1] to put all these ApiCallback methods in one place.